taskcluster/kinds/docker-image/kind.yml

```diff
@@ -0,0 +1,66 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.docker_image:transforms
+    - taskgraph.transforms.cached_tasks:transforms
+    - taskgraph.transforms.task:transforms
+
+tasks:
+    client-tox-py38:
+        definition: tox
+        args:
+            BASE_IMAGE: python:3.8
+            SETUP_COMMAND: 'apt-get update && apt-get install -y zip xz-utils build-essential libssl-dev libffi-dev python3-dev'
+    client-tox-py39:
+        definition: tox
+        args:
+            BASE_IMAGE: python:3.9
+            SETUP_COMMAND: 'apt-get update && apt-get install -y zip xz-utils build-essential libssl-dev libffi-dev python3-dev'
```

Comment:
There's probably not much benefit in using tox here if we're going to split the tasks out one per Python version anyway. But that can be a follow-up if you want to reduce the simultaneous changes in this PR

Reply:
Right, I was aiming for the minimum amount of changes to reproduce the existing tasks (which is already quite a bit), I agree there's opportunities for later improvements.